packages/application/src/api/http/controllers/authutils.ts

```diff
@@ -0,0 +1,76 @@
+import { transactionSerialize } from '@zondax/filecoin-signing-tools';
+import { AddressSecp256k1 } from 'iso-filecoin/address'
+import { Signature, verify } from '@noble/secp256k1';
+import { blake2b } from '@noble/hashes/blake2';
```

Comment:
we should try and find a signing library that validates a layer above having to thing about the specific signing algorithm